roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -176,3 +176,12 @@ memcached_servers = {{ hostvars|ursula_memcache_hosts(groups, memcached.port) }}
 allow_headers = X-Auth-Token,X-Identity-Status,X-Roles,X-Service-Catalog,X-User-Id,X-Tenant-Id,X-OpenStack-Request-ID,X-Trace-Info,X-Trace-HMAC,OpenStack-Volume-microversion
 expose_headers = X-Auth-Token,X-Subject-Token,X-Service-Token,X-OpenStack-Request-ID,OpenStack-Volume-microversion
 allow_methods = GET,PUT,POST,DELETE,PATCH
+
+{% if barbican.enabled|bool %}
```

Comment:
Good idea, I will add the preflight checks, `cinder.encryption.enabled` and I'll also do `nova.volume_encryption.enabled`.